app/grandchallenge/cases/views.py

```diff
@@ -346,6 +346,15 @@ class ImageSearchResultView(
     def get(self, request, *args, **kwargs):
         qs = self.get_queryset()
         prefixed_interface_slug = request.GET.get("prefixed-interface-slug")
+        interface = get_object_or_404(
+            ComponentInterface,
+            slug=prefixed_interface_slug.replace(
+                INTERFACE_FORM_FIELD_PREFIX, ""
+            ),
```

Comment:
What if there is no prefixed_interface_slug. This would result in a server error. Should we handle it more gracefully?